query-engine/query-engine/src/server/mod.rs

```diff
@@ -64,8 +61,7 @@ pub async fn listen(opts: PrismaOpt) -> PrismaResult<()> {
     app.at("/server_info").get(server_info_handler);
     app.at("/status").get(|_| async move { Ok(json!({"status": "ok"})) });
 
-    info!("Started http server on {}:{}", addr.ip(), addr.port());
-    app.listen(addr).await?;
+    app.listen((&*opts.host, opts.port)).await?;
```

Comment:
Fixed this by adding the `tracing-log` bridge.

Reply:
Actually we don't need `tracing-log` since it's enabled by default; going to patch `tide` to include the port in both the string and the structured message. We should also patch the bridge from tracing and log.